mlir/include/mlir/Dialect/Rock/utility/loweringUtils.h

```diff
@@ -40,6 +40,10 @@ struct RegsAsMatrixSubTiles {
   // Using just a iterative dix, this provides access to threadwise sub-tile
   // of the matrix.
   ArrayAttr threadSubTile;
+  // This is a [tid] to a 2D subtile view.
+  // i.e. [tid] --> [m_tid, n_tid]
+  // where |m_tid| x |n_tid| == workgroup size.
```

Comment:
Add: "It is equivalent to removing all `iter`-dependent components from `blockSubTile`."